kubernetes/gke-base/cfg/client-console-pod.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
 spec:
   containers:
   - name: client-console
-    image: docker.io/confluentinc/cp-server-operator:5.4.1.0
+    image: docker.io/confluentinc/cp-server-operator:5.5.0.0
```

Comment:
Should this be parameterized as `OPERATOR_CP_IMAGE_TAG`?

Reply:
@ybyzek please see the comment in the PR description and git commit msg.

IIUC it is not straightfoward to use a env var in a K8s definition or on the CLI with `kubectl`.  I'm investigating, but wanted to get the branch caught up for now.

Reply:
Ack. Unless it can be tied to the parameter, does that mean we will always be playing catchup?

Reply:
It just means we have to manually manage this until I determine a way to leverge the config.env value.  I'm investigating and will update to that in another PR, and I won't close this issue.  I just wanted to get the branch "up to date" with this.